ui/desktop/src/components/ViewRecipeModal.tsx

```diff
@@ -102,28 +101,34 @@ export default function ViewRecipeModal({ isOpen, onClose, config }: ViewRecipeM
     }
   }, [isOpen, getExtensions, recipeExtensions, extensionsLoaded]);
 
-  // Use effect to set parameters whenever instructions or prompt changes
+  // Only auto-parse parameters from instructions if no parameters exist yet
+  // This prevents overwriting existing parameter configurations
```

Comment:
I'm not sure I am following the code here, but does this mean once you set parameters, we no longer update the parameters when the prompt changes